README.md

```diff
@@ -145,14 +145,15 @@ export default Ember.Route.extend({
 ```javascript
 // view/application.js
 import Ember from 'ember';
-import DecorateMixin, { computedDecorate } from 'ember-clothier/decorate-mixin';
+import { computedDecorateWithSetter } from 'ember-clothier/decorate-mixin';
 
-export default Ember.View.extend(DecorateMixin, {
-  activatables: computedDecorate('model', 'activatable'),
+export default Ember.View.extend({
+  model: computedDecorateWithSetter('activatable'),
```

Comment:
Maybe `decoratorFacotry` as we don't decorating (`decorate`) something in this step and doing it later depends on code/user decision.